src/main/java/org/shanerx/tradeshop/Utils.java

```diff
@@ -123,4 +144,10 @@ public boolean containsAtLeast(Inventory inv, Material mat, int amt) {
 		}
 		return count >= amt;
 	}
-}
\ No newline at end of file
+	
+	public String colorize(String msg)
+	{
+		msg = ChatColor.translateAlternateColorCodes('&', msg);
+		return msg;
+	}	
+}
```

Comment:
Can't believe I haven't wrapped this up earlier. We'll have to replace all calls of ChatColor.translateAlternateColorCodes in any class.